pkgs/tools/networking/nfdump/default.nix

```
@@ -1,19 +1,33 @@
-{ stdenv, fetchFromGitHub, bzip2, yacc, flex }:
+{ stdenv, fetchFromGitHub
+, autoconf, automake, libtool, pkg-config
+, bzip2, libpcap, flex, yacc }:
 
-let version = "1.6.16"; in
+let version = "1.6.17"; in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   name = "nfdump-${version}";
 
   src = fetchFromGitHub {
     owner = "phaag";
     repo = "nfdump";
     rev = "v${version}";
-    sha256 = "0dgrzf9m4rg5ygibjw21gjdm9am3570wys7wdh5k16nsnyai1gqm";
+    sha256 = "1z8zpvd9jfi2raafcbkykw55y0hd4mp74jvna19h3k0g86mqkxya";
   };
 
-  nativeBuildInputs = [yacc flex];
-  buildInputs = [bzip2];
+  nativeBuildInputs = [ autoconf automake flex libtool pkg-config yacc ];
+  buildInputs = [ bzip2 libpcap ];
+
+  preConfigure = ''
+    patchShebangs autogen.sh
+    LIBTOOLIZE=libtoolize ./autogen.sh
```

This should be enough:
```suggestion
    ./autogen.sh
```

---

This is just a workaround for darwin. See https://github.com/phaag/nfdump/blob/master/autogen.sh#L5-L7. I can add a condition around it if you wish.

---

I don't think we need a condition in that case but you could add the comment here stating why it is necessary.